drinfomon.lic

```diff
@@ -553,41 +553,43 @@ sleep(0.1) until Char.name && !Char.name.empty?
-  "Muspar'i"					=> ["[[Old Lata'arna Keep, Teller Windows]]"],
-  'Ratha'						  => ['[[Lower Bank of Ratha, Cashier]]', '[[Sshoi-sson Palace, Grand Provincial Bank, Bursarium]]'],
-  "Raven's Point"				=> ["[[Bank of Raven's Point, Depository]]"],
-  'Riverhaven'				=> ['[[Bank of Riverhaven, Teller]]'],
-  "Rossman's Landing"	=> ['Unknown Description'],
-  'Shard'						  => ["[[First Bank of Ilithi, Teller's Windows]]"],
-  'Therenborough'			=> ['[[Bank of Therenborough, Teller]]'],
-  'Throne City'				=> ['[[Faldesu Exchequer, Teller]]']
+  "Ain Ghazal"         => ["Unknown Description"],
+  "Boar Clan"          => ["Unknown Description"],
+  "Crossings"          => ["[[Provincial Bank, Teller]]"],
+  "Dirge"              => ["[[Dirge, Traveller's Bank]]"],
+  "Fang Cove"          => ["[[First Council Banking, Vault]]"],
+  "Hibarnhvidar"       => ["[[Second Provincial Bank of Hibarnhvidar, Teller]]", "[[Hibarnhvidar, Teller Windows]]", "[[First Arachnid Bank, Lobby]]"],
+  "Ilaya Taipa"        => ["Unknown Description"],
+  "Leth Deriel"        => ["[[Imperial Depository, Domestic Branch]]"],
+  "Ratha"              => ["[[Lower Bank of Ratha, Cashier]]", "[[Sshoi-sson Palace, Grand Provincial Bank, Bursarium]]"],
+  "Riverhaven"         => ["[[Bank of Riverhaven, Teller]]"],
+  "Shard"              => ["[[First Bank of Ilithi, Teller's Windows]]"],
+  "Therenborough"      => ["[[Bank of Therenborough, Teller]]"],
+  "Throne City"        => ["[[Faldesu Exchequer, Teller]]"],
+  "Aesry Surlaenis'a"  => ["[[Tona Kertigen, Deposit Window]]"],
+  "Chyolvea Tayeu'a"   => ["[[Chyolvea Tayeu'a, Teller]]"],
+  "Hara'jaal"          => ["Unknown Description"],
+  "Mer'Kresh"          => ["[[Harti Clemois Bank, Teller's Window]]"],
+  "Muspar'i"           => ["[[Old Lata'arna Keep, Teller Windows]]"],
+  "Raven's Point"      => ["[[Bank of Raven's Point, Depository]]"],
+  "Rossman's Landing"  => ["Unknown Description"]
 }
 
 vault_titles = {
-  'Crossings'					=> ['[[Crossing, Carousel Chamber]]'],
-  'Leth Deriel'				=> ['[[Leth Deriel, Carousel Chamber]]'],
-  "Muspar'i"					=> ["[[Muspar'i, Carousel Square]]"],
-  'Ratha'						=> ['[[Ratha, Carousel Square]]'],
-  'Riverhaven'				=> ['[[Riverhaven, Carousel Chamber]]'],
-  'Shard'						=> ['[[Shard, Carousel Square]]'],
-  "Mer'Kresh"					=> ["[[Mer'Kresh, Carousel Square]]"],
-  'Therenborough'				=> ['[[Therenborough, Carousel Square]]']
+  "Crossings"          => ["[[Crossing, Carousel Chamber]]"],
+  "Fang Cove"          => ["[[Fang Cove, Carousel Chamber]]"],
+  "Leth Deriel"        => ["[[Leth Deriel, Carousel Chamber]]"],
+  "Ratha"              => ["[[Ratha, Carousel Square]]"],
+  "Riverhaven"         => ["[[Riverhaven, Carousel Chamber]]"],
+  "Shard"              => ["[[Shard, Carousel Square]]"],
+  "Therenborough"      => ["[[Therenborough, Carousel Square]]"],
+  "Mer'Kresh"          => ["[[Mer'Kresh, Carousel Square]]"],
+  "Muspar'i"           => ["[[Muspar'i, Carousel Square]]"]
```

Comment:
* Sorted entries alphabetically
* Standardized on double-quote rather than some entries with single and some with double
* Added Fang Cove
* Replaced tabs with spaces